yuniql-platforms/postgresql/Yuniql.PostgreSql.csproj

```diff
@@ -21,7 +21,7 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-    <DocumentationFile>C:\play\yuniql\yuniql-platforms\postgresql\Yuniql.PostgreSql.xml</DocumentationFile>
+    <DocumentationFile>Yuniql.PostgreSql.xml</DocumentationFile>
```

Comment:
Thanks! :)

Reply:
Yup, thought to make it separate but then kept it here. Can move this change to separate ISSUE/PR etc